Update bookings-in-outlook.md (#9473)
MicrosoftDocs/microsoft-365-docs: microsoft-365/bookings/bookings-in-outlook.md

1. Bookings with me and Bookings share the same licensing model. However, Bookings doesn't have to be turned on for the organization using tenant settings for users to access Bookings with me. The Bookings app must be enabled for users to have access to Bookings with me.

   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).

2. Calendar FreeBusy Anonymous sharing must be enabled to use Bookings with me. This allows the Bookings page to have access to the free/busy information in your Outlook calendar. Use PowerShell to check the status.

   ```PowerShell
     Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
   ```

>     "Anonymous:CalendarSharingFreeBusySimple" should be one of the domains in the response.

Reviewer: ```suggestion
    "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or  CalendarSharingFreeBusyReviewer (default).
```

>     "Anonymous:CalendarSharingFreeBusySimple"" should be one of the domains in the response.

Reviewer 2: Contains an extra closing quote.

Author: Thank you. this has been fixed

   To enable anonymous sharing, use the following command.

   ```PowerShell

>      Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}
>    ```

Reviewer: ```suggestion
     Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}
   ```

For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy).
```

> 

Reviewer: For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy).

Author: Made the changes. Thank you.

1. Bookings with me and Bookings share the same licensing model. However, Bookings doesn't have to be turned on for the organization using tenant settings for users to access Bookings with me. The Bookings app must be enabled for users to have access to Bookings with me.

   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).

2. Calendar FreeBusy Anonymous sharing must be enabled to use Bookings with me. This allows the Bookings page to have access to the free/busy information in your Outlook calendar. Use PowerShell to check the status.

   ```PowerShell
     Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
   ```

>     "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or  CalendarSharingFreeBusyReviewer (default).

Reviewer 3: ```suggestion
    "Anonymous:<SharingPolicyAction>" must be one of the domains in the response. <SharingPolicyAction> value can be CalendarSharingFreeBusySimple, CalendarSharingFreeBusyDetail, or CalendarSharingFreeBusyReviewer (default).
```

   To enable anonymous sharing, use the following command.

   ```PowerShell

>      Set-SharingPolicy "Default Sharing Policy" -Domains @{Add="Anonymous:CalendarSharingFreeBusySimple"}

Reviewer 4: Why not using the default of new teants: "CalendarFreeBusyReviewer"

Reviewer: Due to the least privilege principle. If anonymous sharing is disabled, this is the minimum requirement to enable it.
Both issue raisers are complaining about allowing additional permissions to anonymous.

Author: Thank you @dariomws  and @ekaplanb

   ```
  

>   For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy)

Reviewer: ```suggestion
  For more information, see [Set-SharingPolicy](/powershell/module/exchange/set-sharingpolicy).
```
